chemprop/nn/transforms.py

```diff
@@ -0,0 +1,63 @@
+from abc import abstractmethod
+
+import torch
+from numpy.typing import ArrayLike
+from sklearn.preprocessing import StandardScaler
+from torch import Tensor, nn
+
+from chemprop.data.collate import BatchMolGraph
+
+
+class TensorTransformBase(nn.Module):
```

Comment:
```suggestion
class _ScaleTransformMixin(nn.Module):
```
IMO the `TensorTransform` is an extremely minimal definition:

```
forward : Tensor -> Tensor
```

but for our purposes we're doing scaling/unscaling, so they should be two separate classes: one that defines the base method signature and one that has some mixin utility